src/components/MenuButton/MenuButtonItem.tsx

```diff
@@ -26,11 +26,22 @@ const getIconClasses = (isDisabled: boolean) => {
 
 const getIconColumn = (data: any, disabled: boolean) => {
   const Icon = data && data.yamUIIcon;
-  return Icon ? (
-    <FixedGridColumn fixed={true} className={getIconClasses(disabled)}>
-      <Icon size={IconSize.MEDIUM} block={true} />
-    </FixedGridColumn>
-  ) : null;
+  const Illustration = data && data.yamUIIllustration;
+  if (Icon) {
+    return (
+      <FixedGridColumn fixed={true} className={getIconClasses(disabled)}>
+        <Icon size={IconSize.MEDIUM} block={true} />
+      </FixedGridColumn>
+    );
+  } else if (Illustration) {
+    return (
+      <FixedGridColumn fixed={true} className={getIconClasses(disabled)}>
```

Comment:
Do we need the getIconClasses call?

Reply:
I suppose not since Illustrations can't have their color changed and thus can't really be disabled. I'll remove this call.